wagtailorderable/signal.py

```diff
@@ -0,0 +1,11 @@
+from django.dispatch import Signal
+
+# queryset is the whole "base queryset": it does not filter only on those updated
+# but filters on a parent key for exemple. eg. Book -> Chapter: Chapters need to be ordered
+# by book, and their ordering is important (can not be shared between all books)
+# in that case, you should use a "child model admin" which will allow you to display (and reorder)
+# only chapters for a specific books. The queryset sends to signals is filtered with the
+# "current book".
+
+pre_reorder = Signal(providing_args=['from_order', 'to_order', 'queryset'])
```

Comment:
Seems `providing_args` is deprecated in Django 3.2. Could you update that to match the new version?

Reply:
I've removed it and added comments instead, similar how wagtail does it in its code.

See #35